docs/dev/multipath.md

```diff
@@ -0,0 +1,244 @@
+     <host mac='<SAN mac from point (1)>' name='SAN' ip='192.168.100.125'/>
+     <host mac='<SNO-SAN mac from point (2)>' name='SNO-SAN' ip='192.168.100.126'/>
+     ```
+
+   - ```bash
+     sudo virsh net-destroy SAN-network
+      ```
+   - ```bash
+     sudo virsh net-start SAN-network
+     ```
+   - Check that the changes updated:
+     ```bash
+     sudo virsh net-dumpxml SAN-network
+     ```
+     example output:
+      ```xml
+        <network connections='2'>
+          <name>SAN-network</name>
+          <uuid>8cb8c0ea-a53c-4bc8-9724-89ad559b0604</uuid>
+          <bridge name='virbr2' stp='on' delay='0'/>
+          <mac address='52:54:00:1d:8f:ce'/>
+          <domain name='SAN-network'/>
+          <ip address='192.168.100.1' netmask='255.255.255.0'>
+            <dhcp>
+              <range start='192.168.100.128' end='192.168.100.254'/>
+              <host mac='52:54:00:58:74:d1' name='SAN' ip='192.168.100.125'/>
+              <host mac='52:54:00:09:e4:57' name='SNO-SAN' ip='192.168.100.126'/>
+            </dhcp>
+          </ip>
+        </network>
+     ```
+5. Click "finish" in points 2-3. 
+6. In SNO-SAN VM -
+    ```bash
+    iscsiadm --mode discovery --type sendtargets --portal 192.168.100.125:3260
+    iscsiadm -m node -T iqn.2009-02.com.example:<iSCSI target you created in SAN> -p 192.168.100.125:3260 -l
+    ```
+6. Run `multipath -ll` to confirm that multipathing is active.
+
+    example output:
+   ```
+   [root@sno-san ~]# multipath -ll
+    mpatha (360000000000000000e00000000010001) dm-0 IET,VIRTUAL-DISK
+    size=120G features='0' hwhandler='0' wp=rw
+    |-+- policy='service-time 0' prio=1 status=active
+    | `- 6:0:0:1 sda 8:0  active ready running
+    `-+- policy='service-time 0' prio=1 status=enabled
+    `- 7:0:0:1 sdb 8:16 active ready running
+    ```
+
+## 2nd approach
+
+In this approach, your beaker machine will act as the target, while the initiator will be the test-infra VM (running on the beaker machine).
+
+1. SSH into your beaker machine and navigate to the directory where the test-infra is cloned.
+2. Create a VM that includes two different networks:
+   ```bash
+   deploy_nodes_with_networking MASTERS_COUNT=1
+   ```
+3. On the target side (beaker machine):
+   - create disk on hypervisor:
+     ```bash
+     qemu-img create -o preallocation=full -f qcow2 /tmp/disk0 30G
+     ```
+   - Install tragetCli package:
+     ```bash
+     dnf install targetcli -y
+     ```
+     ```bash
+     targetcli clearconfig  confirm=True
+     targetcli backstores/fileio create name=disk0 size=30G file_or_dev=/tmp/disk0
+      ```
+   - Create a target:
+     ```bash
+     targetcli /iscsi create iqn.2023-01.com.example:target01
+     ```
+   - Create a LUN:
+     ```bash
+     targetcli /iscsi/iqn.2023-01.com.example:target01/tpg1/luns create /backstores/fileio/disk0
+     ```
+   - Create portal:
+     ```bash
+     targetcli /iscsi/iqn.2023-01.com.example:target01/tpg1/portals create <1st tearget IP>
```

Comment:
s/1st tearget IP/1st target IP
Where do you take the IP from? Is it the IP of the bridge the VM is connected to?

Reply:
@AlonaKaplan You can view the networks added to your beaker machine (which is the target) after running "make deploy_nodes_with_networking MASTERS_COUNT=1." To find the appropriate IP for each network, run "ip a" and search for the relevant network .

Reply:
I guess `ip a` on a beaker server provides a long output.  Can you please add more detailed instructions on how you "search for the relevant network".

Reply:
@AlonaKaplan  for example:
![Screenshot from 2024-07-02 12-46-34](https://github.com/openshift/assisted-service/assets/58630993/220a1e21-44cd-43df-a529-79ca6790876c)

Reply:
I think it is the tap device, it's pretty weird it has an IP. I think the bridge IP should be used and not the tap one... But if it works maybe it doesn't matter, idk.

Reply:
@AlonaKaplan isn't the bridge IP is the one indicated by inet?

Reply:
I don't think it is the bridge, the bridge name usually has "virtbr" prefix.